server/api/reviews.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express')
+const router = express.Router()
+
+const {
+    models: { User, Review, Product },
+} = require('../db')
+
+router.get('/', async (req, res, next) => {
```

Comment:
You might want to include the User so we are able to get the User information

Reply:
Same for the route below

Reply:
Great, I'll include the Product as well just in case

Reply:
Perfect, good call!